docs/security/securing-your-server.md

```diff
@@ -347,10 +347,10 @@ Here's how to install and configure Fail2Ban:
 3.  Set the `bantime` variable to specify how long (in seconds) bans should last.
 4.  Set the `maxretry` variable to specify the default number of tries a connection may be attempted before an attacker's IP address is banned.
 5.  Press `Control-x` and then press `y` to save the changes to the Fail2Ban configuration file.
-6.  Restart Fail2Ban by using `sudo service fail2ban restart`.
+6.  Restart Fail2Ban by using `sudo /bin/systemctl restart  fail2ban.service`.
```

Comment:
Couldn't this be further simplified as just `systemctl restart fail2ban`? Also, with current SystemD systems like CentOS  does the service also need to be enabled to be reboot persistent?